_data/simple-icons.json

```diff
@@ -9257,7 +9261,7 @@
         {
             "title": "Tokyo Metro",
             "hex": "149DD3",
-            "source": "https://en.wikipedia.org/wiki/File:TokyoMetro.svg"
+            "source": "https://en.wikipedia.org/wiki/File:Tokyo_Metro_combined_logo.svg"
```

Comment:
Could be the wikimedia link:
https://commons.wikimedia.org/wiki/File:Tokyo_Metro_combined_logo.svg